ERR_PNPM_MISSING_TIME is not actionable

### Verify latest release

- [x] I verified that the issue exists in the latest pnpm release

### pnpm version

v10.16.0

### Which area(s) of pnpm are affected? (leave empty if unsure)

Dependencies resolver

### Link to the code that reproduces this issue or a replay of the bug

_No response_

### Reproduction steps

0. `corepack use pnpm`
1. Add `@eslint/js` to your dependencies:
```
	"devDependencies": {
		"@eslint/js": "9.35.0",
    }
```
2. add `minimumReleaseAge: 1440` in the `pnpm-workspace.yaml` file
3. run `pnpm up -L`


Same problem on
```
@testing-library/dom
@testing-library/jest-dom
@testing-library/react
@testing-library/user-event
web-vitals
react-scripts
@tsconfig/next
@types/hast
@types/eslint-plugin-jsx-a11
...and more
```

### Describe the Bug

It prints:
```
 ERR_PNPM_MISSING_TIME  The metadata of @eslint/js is missing the "time" field

This error happened while installing a direct dependency of /my-app

If you cannot fix this registry issue, then set "resolution-mode" to "highest".
```

### Expected Behavior

1. it doesn't suggest me to change resolution mode, as this will not help
2. it suggests doing something else, like adding the package to the ignored list, so that I can fix this
3. ignore the package automatically?

### Which Node.js version are you using?

v24.7.0

### Which operating systems have you used?

- [ ] macOS
- [ ] Windows
- [x] Linux

### If your OS is a Linux based, which one it is? (Include the version if relevant)

Arch, EndeavorOS

I don't think that this issue is fully fixed. @zkochan @MonstraG 

If you run the same commands in interactive mode (pnpm up -L -r -i), you will still get the ERR_PNPM_MISSING_TIME exception, even after pruning the store. Without interactive mode, it works, but pulls in a few alpha/beta versions. 

If you try the non-interactive mode after try to update a package with interactive mode, you will get the same error again until pruning the store again.

I assume that interactive mode is storing the metadata differently than non-interactive mode, which then "corrupts" the store. It looks like up --interactive is already pulling in full metadata (https://github.com/pnpm/pnpm/blob/main/reviewing/outdated/src/outdatedDepsOfProjects.ts#L40), but I don't know the codebase well enough to really analyze the issue.

--- a/.changeset/tiny-shoes-brush.md
+++ b/.changeset/tiny-shoes-brush.md
@@ -1,0 +1,6 @@
+---
+"@pnpm/constants": patch
+"pnpm": patch
+---
+
+The full metadata cache should be stored not at the same location as the abbreviated metadata. This fixes a bug where pnpm was loading the abbreviated metadata from cache and couldn't find the "time" field as a result [#9963](https://github.com/pnpm/pnpm/issues/9963).

--- a/packages/constants/src/index.ts
+++ b/packages/constants/src/index.ts
@@ -15,7 +15,7 @@ export const WORKSPACE_MANIFEST_FILENAME = 'pnpm-workspace.yaml'
 // of one package/version
 export const ABBREVIATED_META_DIR = 'metadata-v1.3'
 export const FULL_META_DIR = 'metadata-full-v1.3' // This is currently not used at all
-export const FULL_FILTERED_META_DIR = 'metadata-v1.3'
+export const FULL_FILTERED_META_DIR = 'metadata-ff-v1.3'
 
 export const USEFUL_NON_ROOT_PNPM_FIELDS = ['executionEnv'] as const
 

